website/refasterrules.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
-title: Refaster templates
+title: Refaster Rules
```

Comment:
```suggestion
title: Refaster rules
```

If we want to match the previous casing. Also below, if so.

Reply:
Hmm, we also have Bug Patterns, so Refaster Rules would be inline with that 🤔. 

(IMO it also looks nicer on the website 😬)

Reply:
Agreed. Let's keep it like this. :)